src/pages/HomeParticipant.vue

```diff
@@ -86,6 +87,47 @@
     :consent-type="consentType"
     @accepted="updateConsent"
   />
+  <div>
+    <PvDialog
+      v-if="isAdobe"
+      v-model:visible="isAdobe"
+      :pt="{ mask: { style: 'backdrop-filter: blur(2px);' } }"
+      :draggable="false"
+      :modal="false"
+      class="border-1h-3rem bg-white"
+    >
+      <template #container>
+        <div v-if="!docCreated" class="bg-white p-3 border-1 border-round w-full">
+          <h2>Please enter your email to proceed</h2>
+          <div class="p-10 flex justify-content-center w-full">
+            <!-- <span class="p-float-label"> -->
+            <PvInputText
+              id="signer-email"
+              v-model="signerEmail"
+              class="w-10"
+              style="width: 25rem"
+              placeholder="Email"
+              type="email"
+              enabled
+            />
+          </div>
+          <div class="flex justify-content-center">
+            <PvButton
+              label="Done"
+              class="no-underline bg-primary mt-2 border-none border-round p-3 w-5 text-white hover:bg-red-900"
+              @click="createConsent"
+            />
+          </div>
+        </div>
+        <div v-else class="bg-white p-3 border-1 border-round w-full">
+          <h2>Please review your email to sign the document and proceed</h2>
+          <div class="justify-content-center flex">
+            <img src="../assets/signDoc2.webp" style="width: 400px; border-radius: 0.5rem" />
+          </div>
+        </div>
+      </template>
+    </PvDialog>
+  </div>
```

Comment:
Can you move this to a separate `AdobeSignDialog.vue` component?

Reply:
Yes!